packages/presentation/presentation_publishing/interfaces/fetch/publishes_unified_search.ts

```diff
@@ -10,22 +10,25 @@ import { BehaviorSubject } from 'rxjs';
 import { TimeRange, Filter, Query, AggregateQuery } from '@kbn/es-query';
 import { useEffect, useMemo } from 'react';
 import { PublishingSubject } from '../../publishing_subject';
+import { PublishesFilters } from '../publishes_filters';
 
-export interface PublishesTimeRange {
+export interface PublishesTimeslice {
+  timeslice$?: PublishingSubject<[number, number] | undefined>;
+}
```

Comment:
This will be used for the time slider control, once it is refactored - separating it out in preparation for that